ui/nuxeo-button-styles.js

```diff
@@ -49,6 +49,19 @@ const template = html`
           @apply --nx-button-secondary-disabled;
         }
 
+        paper-button.text {
+          @apply --nx-button-text;
+        }
+
+        paper-button.text:hover,
+        paper-button.text:focus {
+          @apply --nx-button-text-hover;
```

Comment:
do we want the same style on both states?

Reply:
This is a good question to be checked with Carolina. This is consistent with what we have above for primary and secondary buttons, though.

Reply:
The [guidelines](https://nuxeowiki.atlassian.net/wiki/spaces/DT/pages/1172767001/On+default+theme#Text-button) don't specify a `focus` style.
Yep, just tried to keep it consistent.